### PR TITLE
Update list-of-terms.md

### PR DESCRIPTION
**Pull request by justinegeffen**
Removed extraneous bolding from headings.

### CODE REVIEW

**Review by natalie-hub — DISMISSED** [review]
These look good, too! 

Who would be the DRI or AOR to add usage examples of some of the terms. For example, Dark Actions makes sense, but we don't use it frequently and I wonder if that's because people aren't sure how to use it.

**Comment by justinegeffen** [comment]
Great question, @natalie-hub, and I am not really sure. Added @michaelschiffmm to this as maybe he would know. :)

**Comment by michaelschiffmm** [comment]
> Great question, @natalie-hub, and I am not really sure. Added @michaelschiffmm to this as maybe he would know. :)

BizOps handles all of the terms and definitions around metrics definitions and usage. Not really anything on terms around "company culture" if you will. I would think that would be HR or MLT.

**Comment by justinegeffen** [comment]
@natalie-hub, I'm going to merge this PR and create a new one with examples for some terms.

File: company/about-mattermost/list-of-terms.md [hunk 1]
```diff
@@ -50,27 +50,27 @@ Example: Giving someone instructions on how to set up OneLogin for MFA on the co
 
 Dark actions create [false openness](list-of-terms.md#false-openness). [Open actions](list-of-terms.md#open-actions) are highly preferred.
 
-### **Dead Tarzan**
+### Dead Tarzan
 
-Discarding an imperfect solution without a clearly thought out and working alternative. Based on idea of [Tarzan of the Jungle](https://en.wikipedia.org/wiki/Tarzan) letting go of a vine without having a new vine to swing to.
+Discarding an imperfect solution without a clearly thought out and working alternative. Based on idea of [Tarzan of the Jungle (https://en.wikipedia.org/wiki/Tarzan) letting go of a vine without having a new vine to swing to.
```
Comment (amyblais) [inline company/about-mattermost/list-of-terms.md]:
```suggestion
Discarding an imperfect solution without a clearly thought out and working alternative. Based on idea of [Tarzan of the Jungle](https://en.wikipedia.org/wiki/Tarzan) letting go of a vine without having a new vine to swing to.
```

**Review by amyblais — APPROVED** [review]

**Pull request merged**